### PR TITLE
Add BSC and SKALE testnet chain configs

### PR DESCRIPTION
**Pull request by ianrowan**
## Link to Issue
Closes: #10038 

## Description of Changes
- Adds deployed contracts for the above chains

## Other Considerations

- RPCs will still need to be added via platform for specific envs

### CODE REVIEW

File: libs/evm-protocols/src/common-protocol/chainConfig.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export enum ValidChains {
   Optimism = 10,
   Mainnet = 1,
   Arbitrum = 42161,
+  BSC = 56,
+  SKALE = 974399131,
```
Comment (timolegros) [inline libs/evm-protocols/src/common-protocol/chainConfig.ts:12]: This chain id is for `SKALE Calypso Hub Testnet`. We should name this something more accurate like: `SKALE_CALYPSO_TESTNET`

**Review by timolegros — CHANGES_REQUESTED** [review]
Per CI errors, you need to update the EventRegistry with the relevant contract/event details. Also, given the new EventRegistry setup, the ChainNodes must be created _before_ the chainConfig is added.

File: libs/evm-protocols/src/common-protocol/chainConfig.ts (hunk 1 above)
Comment (ianrowan) [reply libs/evm-protocols/src/common-protocol/chainConfig.ts]: Should be good to use SKALE_TEST as we're only going to use this for testnet then replace with mainnet. Also updated event registry

**Review by dillchen — APPROVED** [review]

**Review by kurtisassad — APPROVED** [review]

**Review by dillchen — APPROVED** [review]

**Review by timolegros — APPROVED** [review]
Just make sure the Chain nodes are added to the db before deploying. Approving on my phone so can't check DB. If the chain node doesn't exist chain-events will fail.

**Pull request merged**